src/CommunityPatch/Patches/Perks/Cunning/Roguery/SlaveTraderPatch.cs

```diff
@@ -0,0 +1,82 @@
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using HarmonyLib;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.Core;
+using TaleWorlds.Localization;
+using static CommunityPatch.HarmonyHelpers;
+
+namespace CommunityPatch.Patches.Perks.Cunning.Roguery {
+
+  public sealed class SlaveTraderPatch : PatchBase<SlaveTraderPatch> {
+    
```

Comment:
```suggestion
    private const float BonusRansomGoldPercent = .20f;

```

Reply:
I think we can do this pattern in bulk to many perks in a refactor PR later.